ui/packages/consul-ui/app/components/consul/upstream-instance/list/index.hbs

```diff
@@ -40,19 +40,19 @@
       {{#if item.LocalBindSocketPath}}
 {{#let (if item.LocalBindSocketMode
           (hash
-            label=(concat '(Local bind mode)')
-            value=(concat '(Mode:' item.LocalBindSocketMode ')')
+            label='Local bind mode'
+            value=item.LocalBindSocketMode
           )
           (hash
-            label=""
-            value=""
+            label='Local bind mode'
+            value="-"
```

Comment:
I don't think we need to do any of this `let`ing now we are always showing a label. We should remove this and just do it inline instead.

Reply:
Refactored!